src/SkillService.Data/Interfaces/IUserSkillRepository.cs

```diff
@@ -11,6 +11,8 @@ public interface IUserSkillRepository
   {
     Task<List<Guid>> GetAsync(Guid userId);
 
+    Task<List<DbUserSkill>> GetUserSkillsAsync(Guid userId);
```

Comment:
please, rename your method to GetAsync, and set GetUserSkillIdsAsync name to method on 12 line